### PR TITLE
add kubeVersion to Chart.yaml

### PR DESCRIPTION
**Pull request by morvencao**
Add `kubeVersion` to Istio chart to specify compatible Kubernetes versions.

### CODE REVIEW

File: install/kubernetes/helm/istio-remote/Chart.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 name: istio-remote
 version: 1.1.0
 appVersion: 1.1.0
+kubeVersion: ">=1.10.0"
```
Comment (gyliu513) [inline install/kubernetes/helm/istio-remote/Chart.yaml]: Yes, have same concern as @sdake, what is the case if kubeVersion smaller than 1.10.0?

**Comment by costinm** [comment]
Same as the other comments, and not clear why it's required.

**Comment by morvencao** [comment]
Okay to close this, as `kubeVersion` should be optional in `Chart.yaml`, see: https://github.com/helm/helm/blob/master/docs/charts.md#the-chartyaml-file

**Pull request closed without merging**